chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -1025,26 +1025,30 @@ def dataset_match(datasets, device_path):
                                           expected_return_code=None)
 
     def cleanup_linux_devices(self, test_servers):
-        """
-        Destroy any partitions on the block device.
-
-        Very linux specific code.
-        :return:
-        """
-        if (self.simulator is not None) or (self.linux_devices_exist() is False):
+        if (self.simulator is not None) or (self.linux_devices_exist() is
+                                            False):
             return
 
         first_test_server = test_servers[0]
 
-        # Erase all volumes if the config does not indicate that there is already
-        # a pre-existing file system (in the case of the monitoring only tests).
-        for lustre_device in config['lustre_devices']:
-            if lustre_device['backend_filesystem'] == 'ldiskfs':
-                linux_device = TestBlockDevice('linux', first_test_server['device_paths'][lustre_device['path_index']])
+        def get_device_path(idx):
+            return first_test_server['device_paths'][idx]
 
-                self.execute_simultaneous_commands(linux_device.destroy_commands,
-                                                   [server['fqdn'] for server in test_servers],
-                                                   'clear block device %s' % linux_device)
+        def cleanup_str(x):
+            return 'wipefs -a {}'.format(x)
+
+        device_paths = [
+            get_device_path(x['path_index']) for x in config['lustre_devices']
+            if x['backend_filesystem'] == 'ldiskfs'
+        ]
+
+        desc = 'clear block devices {}'.format(', '.join(device_paths))
+        wipe_commands = map(cleanup_str, device_paths)
+        self.execute_commands(wipe_commands, first_test_server['fqdn'], desc)
```

Comment:
What if the disks aren't fully meshed such as they could/would be if they were only dual-ported?

Reply:
Will have to refactor this further when that becomes the case. Right now these disks are shared across all hosts.

Reply:
The `config` currently tells you which disks are available on which nodes.  Each `lustre_servers` has a list of disks it's connected to.

Reply:
Indeed, but the test config is currently wired to list devices by backing fs type + idx anyway, so there is a baked in presumption that it's shared between all hosts:

```
[
  {
    "backend_filesystem": "zfs",
    "path_index": 0
  },
  {
    "backend_filesystem": "ldiskfs",
    "path_index": 1
  },
  {
    "backend_filesystem": "zfs",
    "path_index": 2
  },
  {
    "backend_filesystem": "ldiskfs",
    "path_index": 3
  },
  {
    "backend_filesystem": "zfs",
    "path_index": 4
  }
]
```

Reply:
Can we open a ticket for this assumption of fully-meshed disk connectivity? At least to point the first person who comes to us with a configuration that isn't fully-meshed at.

Reply:
Opened: https://github.com/intel-hpdd/intel-manager-for-lustre/issues/352 as a general issue on SSI / EFS cleanup (feel free to add to it).